src/components/SelectInput/SelectInput.tsx

```diff
@@ -89,6 +89,12 @@ export function SelectInput<T = SelectInputOption>({ className, placeholder, opt
     setIsCursorInside(false);
   }, []);
 
+  const onKeyUp = useCallback((event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === 'Delete' || event.key === 'Backspace') {
```

Comment:
do we rly want that? If im not mistaken - it will make it impossible for user to fix their input.
For example: I wrote "12356" and want to change it to "12345" - once I click backspace to remove "6" from the end - entire input will be cleared and I'm confused

Reply:
I've added a checking that doesn't delete all text in input, but just clears the selected value.